server/collection/model.ts

```diff
@@ -75,7 +82,7 @@ export class Collection extends Model<
 
 	@Default('choose-best')
 	@Column(DataType.ENUM('none', 'minimal', 'medium', 'choose-best'))
-	readNextPreviewSize!: CreationOptional<ReadNextPreviewSize | null>;
+	readNextPreviewSize!: CreationOptional<ReadNextPreviewSize>;
```

Comment:
I do this once or twice: if the column has a default value I think it's pretty safe to remove the `| null`. 
Technically it can still be set to null manually, e..g `Collection.update({ readNextPreviewSize: null }, { where: ... }), while throwing a type error, will still run, but I couldn't find anywhere where that could possibly happen.